lib/paypal_nvp.rb

```diff
@@ -31,6 +31,17 @@ def initialize(sandbox = false, extras = {})
     end
     @extras = extras
     @rootCA = @rootCA || '/etc/ssl/certs'
+
+
+    if config
+      # Get the timeout options from the config
+      @open_timeout = config[type]["open_timeout"]
+      @read_timeout = config[type]["read_timeout"]
+    else
+      # Get the timeout options from the extra parameter
+      @open_timeout = extras[:open_timeout]
+      @read_timeout = extras[:read_timeout]
+    end
```

Comment:
Good spot. Updated in https://github.com/fac/paypal_nvp/pull/1/commits/8a362442e466c831cc97734f29e07fab92996cfa